qiime2/sdk/plugin_manager.py

```diff
@@ -85,6 +85,15 @@ def _integrate_plugin(self, plugin):
                     " and %r" %
                     (name, record.plugin.name, self.formats[name].plugin.name)
                 )
+
+            fmt = record.format
```

Comment:
Could you add a `#TODO` to remove when sniff is removed? The `Format` would use an ABC at that point.